client/src/AddForm.js

```diff
@@ -1,97 +0,0 @@
-import React, { Component } from "react";
-import "./App.css";
-import axios from "axios";
-
-const BASE_URL = "http://localhost:3005";
-
-class AddForm extends Component {
-  constructor() {
-    super();
-    this.state = {
-      name: "",
-      image_url: "",
-      scary: {
-        level: 0
-      }
-    };
-  }
-  handleScaryLevel = event => {
-    const { scary } = this.state.scary;
-    const newScary = Object.assign({}, scary);
-    newScary.level = event.target.value;
-    this.setState({ scary: newScary });
-  };
-  handleChange = (event, key) => {
-    // console.log("Event : ", event);
-    this.setState({
-      [key]: event.target.value
-    });
-    // const { name, image_url, scary } = this.state;
-    // this.setState({
-    //   name: name || this.state.name,
-    //   image_url: image_url || this.state.image_url,
-    //   scary: scary || this.state.scary
-    // const category = { ...this.state[cat] };
-    // category[key] = event.target.value;
-    // this.setState({ [cat]: category });
-  };
-
-  handleClick = () => {
-    let newCreature = {
-      name: this.state.name,
-      image_url: this.state.image_url,
-      scary: {
-        level: this.state.scary.level
-      }
-    };
-    console.log("Adding to list", newCreature);
-
-    axios.post(`${BASE_URL}/creatures`, newCreature).then(response => {
-      console.log("Posted", response);
-      this.props.updateState(response.data);
```

Comment:
i might be missing, but i don't see where this function is defined.

Reply:
Its defined inline within App.js 

  render() {
    return (
      <div className="App">
        <Add updateState={data => this.setState({ creatures: data })} />
        <h2>Haunted House</h2>
        <Creatures
          creatures={this.state.creatures}
          updateState={data => this.setState({ creatures: data })}
        />
      </div>

Reply:
i see now - my mistake I didn't have the files expanded